Resources/Private/Language/locallang.xlf

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+<xliff version="1.0">
+    <file source-language="en" datatype="plaintext" original="EXT:typo3-crawler-widget/Resources/Private/Language/locallang.xlf" date="2020-08-27T08:00:00Z"
```

Comment:
EXT:typo3-crawler-widget => EXT:crawler_widget (use the official TYPO3 extension key)